src/pages/developer-tools/bower/index.md

```diff
@@ -3,13 +3,23 @@ title: Bower
 ---
 ## Bower
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/developer-tools/bower/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+Bower is a package manager for the web. Bower can manage components that contain HTML, CSS, JavaScript, fonts or even image files. Bower is optimized for the front-end. If multiple packages depend on a package - jQuery for example - Bower will download jQuery just once. This is known as a flat dependency graph and it helps reduce page load.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+#### Install Bower
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+Bower is a command line utility. Install it with npm.
+
+    npm install -g bower
```

Comment:
You can format this command with syntax highlighting by wrapping it in backticks and indicating it's a shell command:


```
```
    ```shell
     npm install -g bower
    ```
```